### PR TITLE
Make sure the Filter view collapses on outside clicks without passing…

### PR DESCRIPTION
**Pull request by dmfs**
… the touch event to the underlying view. Also make the categories focusable allowing to select them with a keyboard. Implements #401

### CODE REVIEW

File: eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/filter/filterstate/Collapsed.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.schedjoules.eventdiscovery.framework.filter.filterstate;
+
+/**
+ * {@link FilterState} decorator which collapses the filter.
+ *
+ * @author Gabor Keszthelyi
```
Comment (lemonboston) [inline eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/filter/filterstate/Collapsed.java]: incorrect author ;)

File: eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/filter/views/EventFilterView.java [hunk 2]
```diff
@@ -183,6 +185,19 @@ public void listen(CategorySelectionChangeListener listener)
     }
 
 
+    private class DismissListner implements PopupWindow.OnDismissListener
```
Comment (lemonboston) [inline eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/filter/views/EventFilterView.java]: a typo in class name

**Review by lemonboston — APPROVED** [review]
Works nicely!
I've also checked on KitKat, works there, too.

File: eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/filter/filterstate/Collapsed.java (hunk 1 above)
Comment (dmfs) [reply eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/filter/filterstate/Collapsed.java]: fixed

File: eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/filter/views/EventFilterView.java (hunk 2 above)
Comment (dmfs) [reply eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/filter/views/EventFilterView.java]: fixed

**Pull request merged**